stHW/SortStation.c

```diff
@@ -0,0 +1,79 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <ctype.h>
+#include <string.h> 
+#include "Stack.h"
+#include "SortStation.h"
+
+int getPrecedence(char op) { 
+    if (op == '-' || op == '+') {
+        return 1;
+    }
+    if (op == '*' || op == '/') {
+        return 2;
+    }
+    return 0; 
+}
+
+char peek(Stack* stack) {
+    if (isEmpty(stack)) {
+        fprintf(stderr, "Attempted to peek at an empty stack\n");
+        exit(EXIT_FAILURE);
+    }
+    return stack->head->value;
+}
+
+char* InfixToPostfix(char* strin) {
```

Comment:
```suggestion
char* infixToPostfix(char* string) {
```
тут что-то странное случилось